code/_onclick/item_attack.dm

```diff
@@ -34,8 +34,10 @@ avoid code duplication. This includes items that may sometimes act as a standard
 	return
 
 /atom/movable/attackby(obj/item/W, mob/user)
-	if(!(W.item_flags & ITEM_FLAG_NO_BLUDGEON))
+	if(W.item_flags & ITEM_FLAG_NO_BLUDGEON)
 		visible_message("<span class='danger'>[src] has been hit by [user] with [W].</span>")
+		return
+	..()
```

Comment:
This directly reverses the previous behavior, is this intended?

Reply:
It's exactly the opposite. Previously, items WITHOUT "no_bludgeon" flag would **not** attack normally.
I simply fixed it.

Reply:
(Resolved on Discord)